### DISCUSSION TITLE
Manually trigger consume directory or via schedule

### DISCUSSION THREAD

**Discussion by KPKoch** [jonaswinkler/paperless-ng#1500] [Feature Requests]
If I bind a persistent Docker volume via NFS, paperlessNG isn't notified of the new file.
To import from this directory, I need to restart the paperlessNG container at whole.
A button to consume the directory NOW or some scheduler which looks every n minutes (30-60) would be a great help.

**Comment by TheFehr** [comment]
There is:
`PAPERLESS_CONSUMER_POLLING=<num>`

You can read more about it here:
https://paperless-ng.readthedocs.io/en/latest/configuration.html
